src/_premake_init.lua

```diff
@@ -1540,6 +1573,7 @@
 	api.alias("fileextension", "fileExtension")
 	api.alias("propertydefinition", "propertyDefinition")
 	api.alias("removefiles", "excludes")
+	api.alias("uacexecutionlevel", "uaclevel")
```

Comment:
I don't see the need for this alias. Let's give it a good name the first time, rather than adding an alias for a name.

Reply:
I was on the fence about this too, since the property is called "uacexecutionlevel" which is quite a mouthful for which "uaclevel" I thought would be a good shorthand, but then again calling the same thing multiple names is bad practice. Shouldn't have included this in the first place. :+1: